src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract double getArea();
```

Comment:
Extract `getArea` to interface and give a proper name to the interface and method. Method should called `calculateArea`, since it calculates, not just gets value from field.

https://howtodoinjava.com/java/basics/java-naming-conventions/